app/src/main/java/org/mozilla/tv/firefox/utils/BuildConstants.kt

```diff
@@ -13,10 +13,14 @@ import org.mozilla.tv.firefox.BuildConfig
 object BuildConstants {
     private const val BUILD_TYPE_DEBUG = "debug"
     private const val BUILD_TYPE_RELEASE = "release"
+    private const val PRODUCT_FLAVOR_GECKO = "gecko"
 
     val isDevBuild: Boolean
         get() = BUILD_TYPE_DEBUG == BuildConfig.BUILD_TYPE
 
     val isReleaseBuild: Boolean
         get() = BUILD_TYPE_RELEASE == BuildConfig.BUILD_TYPE
+
+    val isGeckoBuild: Boolean
+        get() = PRODUCT_FLAVOR_GECKO == BuildConfig.FLAVOR
```

Comment:
Same thing as before.  This might be me being overly inflexible, so feel free to push back on this, but I don't really like the existence of these methods.  It seems like a code smell to me.  I would much rather handle this via source set.

Reply:
@Baron-Severin you do raise a good concern though, i'll create a follow-up story for all debug/test related code?